resource-management/pkg/service-api/endpoints/installer.go

```diff
@@ -123,6 +124,52 @@ func (i *Installer) handleClientUnRegistration(resp http.ResponseWriter, req *ht
 	return
 }
 
+func (i *Installer) NodeHandler(resp http.ResponseWriter, req *http.Request) {
+	klog.V(3).Infof("handle /resource/nodes/. URL path: %s", req.URL.Path)
+
+	switch req.Method {
+	case http.MethodGet:
+		regionName, rpName, nodeId := getNodeId(req)
+		resp.WriteHeader(http.StatusOK)
+		resp.Header().Set("Content-Type", "text/plain")
+
+		region := location.GetRegionFromRegionName(regionName)
+		resourceParition := location.GetPartitionFromPartitionName(rpName)
```

Comment:
In my test environment, the line 137 return error:

ubuntu@ip-172-31-8-82:~/go/src/GRS$ /usr/local/go/bin/go run resource-management/cmds/service-api/service-api.go --master_ip=ec2-52-11-1-193.us-west-2.compute.amazonaws.com --resource_urls=ec2-54-187-22-27.us-west-2.compute.amazonaws.com:9119,ec2-35-91-52-72.us-west-2.compute.amazonaws.com:9119 --enable_metrics=false -v=3
# global-resource-service/resource-management/pkg/service-api/endpoints
resource-management/pkg/service-api/endpoints/installer.go:137:20: assignment mismatch: 1 variable but location.GetPartitionFromPartitionName returns 2 values

Reply:
This error was caused independent implementation and auto code merge. Resolved in PR 148